docs/admin/node-problem.md

```diff
@@ -0,0 +1,199 @@
+---
+---
+
+* TOC
+{:toc}
+
+## Node Problem Detector
+
+*Node problem detector* is a [DaemonSet](/docs/admin/daemons/) monitoring the
+node health. It collects node problems from various daemons and reports them
+to the apiserver as [NodeCondition](/docs/admin/node/#node-condition) and
+[Event](/docs/api-reference/v1/definitions/#_v1_event).
+
+It supports some known kernel issue detection now, and will support more and
+more node problems over time.
+
+Currently Kubernetes won't take any action on the node conditions and events
+generated by node problem detector. In the future, a remedy system will be
+introduced to deal with node problems.
+
+See more information
+[here](https://github.com/kubernetes/node-problem-detector).
+
+## Enable/Disable in GCE cluster
+Node problem detector is running as a cluster
+[addon](docs/admin/cluster-large/#addon-resources) enabled by default in the
+gce cluster.
+
+You can enable/disable it by setting the environment variable
+`KUBE_ENABLE_NODE_PROBLEM_DETECTOR` before `cluster/ kube-up.sh`.
+
+## Other Environment
+To enable node problem detector in other environment outside of GCE, you can use
+either addon pod or `kubectl`.
+
+### Addon Pod (Recommended)
+Create `node-problem-detector.yaml`, and put it under the addon pods directory
```

Comment:
When should you use the addon pod vs kubectl?
- Will not overwrite the default configuration
- Is this recommended for GKE or other managed clusters, or only for manual clusters?